test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -294,13 +294,21 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	},
 	// --
 
-	// k8s.io/kubernetes/pkg/apis/storage/v1alpha1
```

Comment:
For minimal diff, keep alpha first and add beta below?

Reply:
done